modules/modify-data-object/build/modify-data-object/Dockerfile

```diff
@@ -1,4 +1,19 @@
 FROM registry.access.redhat.com/ubi8/ubi-minimal AS builder
-RUN microdnf install -y golang-1.17.* && microdnf clean all
+RUN microdnf install -y tar gzip && microdnf clean all
+ENV TASK_NAME=modify-data-object \
+    GOFLAGS="-mod=vendor"
+WORKDIR /src/${TASK_NAME}
+RUN curl -L https://go.dev/dl/go1.18.3.linux-amd64.tar.gz | tar -C /usr/local -xzf -
```

Comment:
Upgrade to go1.18.4? Can we instead use ubi9?

Reply:
we definitely can. I can do it in follow-up PRs